packages/zoe/src/contractSupport/priceAuthorityQuoteMint.js

```diff
@@ -1,34 +1,23 @@
-import {
-  AssetKind,
-  makeDurableIssuerKit,
-  prepareIssuerKit,
-} from '@agoric/ertp';
-import { makeScalarBigMapStore } from '@agoric/vat-data';
+import { AssetKind, reallyPrepareIssuerKit } from '@agoric/ertp';
+import { provideDurableMapStore } from '@agoric/vat-data';
 
 /**
  *
  * @param {import('@agoric/vat-data').Baggage} baggage
+ * @returns {ERef<Mint<'set'>>}
  */
 export const provideQuoteMint = baggage => {
-  /** @type {ERef<Mint<'set'>>} */
-  let baggageQuoteMint;
-  if (baggage.has(`quoteMintIssuerBaggage`)) {
-    const issuerBaggage = baggage.get(`quoteMintIssuerBaggage`);
-    baggageQuoteMint = /** @type {Mint<'set'>} */ (
-      prepareIssuerKit(issuerBaggage).mint
-    );
-  } else {
-    const issuerBaggage = makeScalarBigMapStore(
-      `scaledPriceAuthority quoteMintIssuerBaggage`,
```

Comment:
Oops! I missed that this uses a different baggage key name.

Must repair.

Reply:
Nevermind. Looking closer, it is only a name for internally tagging the baggage, which it uses to print itself and in error diagnostics. As far as the baggage tree, the old quote only uses `'quoteMintIssuerBaggage'`, so the new code is correct. 

There is an observable difference though: The provide* and prepare* convention uses one name for both purposes, so we lose the separate `'scaledPriceAuthority quoteMintIssuerBaggage'` for baggage born using this new PR. Fortunately, this change caused no tests to fail, so we expect it is inconsequential.

Resolving.